zebra/zebra_vty.c

```diff
@@ -2346,6 +2347,38 @@ DEFUN (show_vrf,
 	return CMD_SUCCESS;
 }
 
+DEFPY (evpn_mh_mac_holdtime,
+	   evpn_mh_mac_holdtime_cmd,
+	   "[no$no] evpn mh mac-holdtime (0-86400)$duration",
+	   NO_STR
+	   "EVPN\n"
+	   "Multihoming\n"
+	   "MAC hold time\n"
+	   "duration in seconds\n")
+{
+	bool set_default;
+
+	set_default = no ? true : false;
+
+	return zebra_evpn_mh_mac_holdtime_update(vty, duration, set_default);
+}
+
+DEFPY (evpn_mh_neigh_holdtime,
+	   evpn_mh_neigh_holdtime_cmd,
+	   "[no$no] evpn mh neigh-holdtime (0-86400)$duration",
+	   NO_STR
+	   "EVPN\n"
+	   "Multihoming\n"
+	   "Neighbor entry hold time\n"
+	   "duration in seconds\n")
+{
+	bool set_default;
+
+	set_default = no ? true : false;
+
+	return zebra_evpn_mh_neigh_holdtime_update(vty, duration, set_default);
```

Comment:
I guess we can handle that inline, without variable assignment, right? 

```
return zebra_evpn_mh_neigh_holdtime_update(vty, duration, no ? true : false);
```

Reply:
ack

Reply:
Cool, thanks, can we do the same for evpn_mh_mac_holdtime_cmd? And I don't have anything more :smile:

Reply:
aah missed that. fixing now.